packaging/QueueNado.spec

```diff
@@ -28,8 +28,8 @@ cd 3rdparty
 unzip -u gtest-1.7.0.zip
 cd ..
 /usr/local/probe/bin/cmake -DCMAKE_CXX_COMPILER_ARG1:STRING=' -fPIC -Ofast -m64 -Wl,-rpath,. -Wl,-rpath -Wl,/usr/local/probe/lib -Wl,-rpath -Wl,/usr/local/probe/lib64 ' -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_CXX_COMPILER=/usr/local/probe/bin/g++
-make
-./UnitTestRunner
+make -j6
```

Comment:
No particular reason. Just better than no -j option. A quick survey of our spec files shows that most use -j8, but others use various different numbers.